Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -2734,6 +2789,25 @@ def search_device_by_ip(raw_res, ip_address):
         raw_res = None
     return raw_res
 
+def enrich_groups(all_group_ids) -> Dict[str, Any]:
+    """
+        Receives a list of group_ids
+        Returns a dict {group_id: group_name}
+    """
+
+    result = dict()
+    for group_id in all_group_ids:
+        params = { 'ids': group_id }
```

Comment:
according to the API, it is possible to send the list of IDs in a single request, instead of a request per ID:
<img width="432" alt="image" src="https://user-images.githubusercontent.com/38749041/234020622-d3eff2dc-a838-4b8b-86b0-004f89eb1607.png">
please check this and update accordingly if this option works, for better performance.